global.json

```diff
@@ -1,17 +1,25 @@
 {
+  "sdk": {
```

Comment:
what is the SDK here?

Reply:
That allows Arcade to install the local SDK so that full framework msbuild can locate it. This was the recommendation from Arcade to help address the issue where a local SDK can't be located (together with the change in configure-toolset.ps1)